app/ui-react/packages/ui/src/Integration/CiCd/CiCdManagePageUI.tsx

```diff
@@ -108,6 +108,7 @@ export class CiCdManagePageUI extends React.Component<
     return (
       <>
         <SimplePageHeader
+          className="pf-u-my-md"
```

Comment:
Same here, maybe we could add a prop similar to how `Button` lets you specify a size prop, I think it'd be good to minimize the user of `className` on components if possible.